src/features/contacts/components/contacts-management/contacts-management.tsx

```diff
@@ -29,6 +31,28 @@ export function ContactsManagement({
   const [searchName, setSearchName] = React.useState("")
   const debouncedSearchName = useDebounce(searchName)
   const { contacts, total } = useContactsList(debouncedSearchName)
+  const inputRef = useRef<HTMLInputElement | null>(null)
+
+  const { updateContact } = useContactsStore()
```

Comment:
I need to doublecheck the behavior if the contact already exists. I think they are overwritten. It would be renaming an existing address with a new nickname, not the other way around (which would be bad).